templates/help-tab/users.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Users Help Tab template.
+ *
+ * @package Activitypub
+ */
+
+?>
+<h2><?php esc_html_e( 'Managing ActivityPub Capabilities', 'activitypub' ); ?></h2>
+
+<p><?php esc_html_e( 'Use the bulk actions on this page to control which users have access to ActivityPub features:', 'activitypub' ); ?></p>
+
+<ol>
+	<li><?php esc_html_e( 'Select the users you want to update by checking the boxes next to their names.', 'activitypub' ); ?></li>
+	<li><?php esc_html_e( 'In the "Bulk Actions" dropdown, choose:', 'activitypub' ); ?>
```

Comment:
Let's replace double and single quotes in this file with entities. There should be prior art for opening and closing double quotes in other files